src/reducers/index.js

```diff
@@ -99,10 +99,18 @@ export const resourceTemplateLoaded = (state, action) => {
 export const makeShortID = () => shortid.generate()
 
 const setRetrieveError = (state, action) => {
-  const resourceTemplateId = action.payload
+  const resourceTemplateId = action.payload.resourceTemplateId
+  const reason = action.payload.reason
   const newState = { ...state }
 
-  newState.editor.serverError = `There was a problem retrieving ${resourceTemplateId}`
+  let serverError = `There was a problem retrieving ${resourceTemplateId}`
+  if (reason) {
+    serverError += `: ${reason}`
+  } else {
+    serverError += '.'
```

Comment:
would it make sense to use something akin to the old verbiage here?  "Sinopia server may be offline or have no resource templates to display" ?

Reply:
Also, vis-a-vis this morning's conversation ... a test for this else branch might be appropriate ...

Reply:
Added tests.

The old verbiage was sometimes wrong. The reducers would specify an error message and this would override and display a different error message.

Reply:
ja, agree and appreciate new logic.  I cleverly changed the old wording to "may" - like, if you don't have a reason, this MIGHT be why.  But not as important as the test - thanks for that.